src/pages/foundation.astro

```diff
@@ -0,0 +1,51 @@
+---
+import Layout from "../layouts/Layout.astro"
+import Heading from "../components/Heading.astro"
+import PostBody from "../components/PostBody.astro"
+
+const title = `DDEV Foundation`
+---
+
+<Layout
+    title={title}
+>
+  <main class="max-w-4xl mx-auto mb-24">
+    <Heading title={`DDEV Foundation`} />
+    <div class="lg:flex">
+      <div class="px-6 lg:px-0 lg:w-2/3">
+        <PostBody>
+          <p>
+            The DDEV Foundation, a Colorado (USA) nonprofit corporation, is the developer of [DDEV](https://ddev.readthedocs.io). It was formerly named the Localdev Foundation. It serves as the fiscal entity for the project and owns the bank account, etc.
+          </p>
+          <p>
+            The DDEV Foundation will be applying for US 501(c)(3) non-profit status in 2023.
+          </p>
+
+          <h2>Where does the money go?</h2>
+
+          <p>
+          DDEV Foundation funds are used for infrastructure costs and to pay part-time and full-time maintainers. Lead Maintainer Randy Fay does not receive any of the funds as his salary is paid by lead sponsor Platform.sh.
+          </p>
+
+          <h2>Contact the DDEV Foundation</h2>
+
+          <p>Contact support (at) ddev.com</p>
+
+          <h2>Snail Mail and Phone:</h2>
+          <pre>
+          DDEV Foundation
+          848 Montclair Dr
+          Palisade, CO 81526
+          USA
+          +1 970-462-7450
+          </pre>
```

Comment:
Why’s this in a code block?

If you definitely want that, you can use Astro’s [`set:html`](https://docs.astro.build/en/reference/directives-reference/#sethtml) to add precisely the text with newlines you want and drop the current indentation that’s a little strange.

But it might as well be a paragraph with `<br>` elements, or you could get fancier and do readers a favor using the [`<address>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address) element.

Reply:
This came over from ddev.foundation.  Your suggestion is welcome.